Intersect.Server/Entities/Entity.cs

```diff
@@ -106,6 +106,9 @@ public string VitalsJson
         [JsonProperty("Vitals"), NotMapped]
         public int[] _vital { get; set; } = new int[(int) Enums.Vitals.VitalCount];
 
+        [JsonIgnore, NotMapped]
+        public int[] _oldvital { get; set; } = new int[(int)Enums.Vitals.VitalCount];
```

Comment:
Rule of thumb, we absolutely should not be naming public variables like this. If it was absolutely necessary it should be `internal`. That said I commented a different solution further along in the review.